integration/networking/resolvconf_test.go

```diff
@@ -134,6 +136,58 @@ func TestInternalNetworkDNS(t *testing.T) {
 	assert.Check(t, is.Contains(res.Stdout(), network.DNSRespAddr))
 }
 
+// Check that '--dns' can be used to name a server inside a '--internal' network.
+// Regression test for https://github.com/docker/for-win/issues/14049
```

Comment:
I'll change the link to a moby ticket when we have one, and add a "fixes" to the description.